src/apify_helpers.js

```diff
@@ -198,7 +229,6 @@ const getActorAdditionalFields = async (z, bundle) => {
             helpText: 'Input configuration for the actor.',
             key: 'inputBody',
             required: false,
-            // TODO: I think this value shouldn't be default, but a prefill (not sure if that's possible)
```

Comment:
It looks that they don't have prefill attribute. I created issue for that in zapier repo https://github.com/zapier/zapier-platform-cli/issues/421